### PR TITLE
Convert stock form exports

### PR DESCRIPTION
**Pull request by benrudolph**
@czue @NoahCarnahan

### CODE REVIEW

**Comment by benrudolph** [comment]
instead of: https://github.com/dimagi/commcare-hq/pull/12877

File: corehq/apps/export/tests/test_export_conversion.py [hunk 1]
```diff
@@ -72,6 +72,30 @@ def test_toggle_turned_on(self, _):
...
+    ('form.transfer.entry.@notme', False),
+    ('form.transfer.@date', True),
+    ('form.transfer.@notme', False),
+
+    ('form.notme.@quantity', False),
+    ('tooshort', False),
+    ('tooshort.tooshort', False),
+    ('', False),
+], TestIsFormStockExportQuestion)
+def test_is_form_stock_question(self, index, expected):
```
Comment (NoahCarnahan) [inline corehq/apps/export/tests/test_export_conversion.py]: Is this function at the wrong indent level? Looks like it is at the same level as `TestIsFormStockExportQuestion`

Comment (NoahCarnahan) [inline corehq/apps/export/tests/test_export_conversion.py]: Oh never mind I see that's how `generate_cases` is supposed to work

**Comment by NoahCarnahan** [comment]
Looks good to me

**Comment by czue** [comment]
one real build failure:
```
======================================================================
ERROR: test_convert_form_export_stock_in_repeat (corehq.apps.export.tests.test_export_conversion.TestConvertStockFormExport)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/vendor/lib/python2.7/site-packages/mock.py", line 1224, in patched
    return func(*args, **keywargs)
  File "/mnt/commcare-hq-ro/corehq/apps/export/tests/test_export_conversion.py", line 680, in test_convert_form_export_stock_in_repeat
    instance, _ = convert_saved_export_to_export_instance(self.domain, saved_export_schema)
  File "/mnt/commcare-hq-ro/corehq/apps/export/utils.py", line 228, in convert_saved_export_to_export_instance
    saved_export.save()
  File "/vendor/lib/python2.7/site-packages/couchdbkit/schema/base.py", line 134, in save
    db.save_doc(doc, **params)
  File "/vendor/lib/python2.7/site-packages/couchdbkit/client.py", line 516, in save_doc
    **params).json_body
  File "/vendor/lib/python2.7/site-packages/restkit/resource.py", line 153, in put
    headers=headers, params_dict=params_dict, **params)
  File "/vendor/lib/python2.7/site-packages/couchdbkit/resource.py", line 140, in request
    response=e.response)
ResourceConflict: Document update conflict.
```
but looks good to me. 

re:

> the more i'm looking at the migration log the more impossible this task seems. i think there are a few bugs in the export generation code in the legacy system that generate indexes that are impossible to map to the new exports and don't yield any data anyways

I think at some point we should just punt and fall back to the remote app-style thing. is that an option?

**Comment by benrudolph** [comment]
> we should just punt and fall back to the remote app-style thing
at first i was hesitant to do that because i was worried about the amount of confused users there would be when they see an "editable" column. perhaps a better way would be to just make it un-editable and it would then make it look fairly normal

**Comment by czue** [comment]
> perhaps a better way would be to just make it un-editable and it would then make it look fairly normal
View changes

yeah this seems like it could be good

**Pull request merged**